### DISCUSSION TITLE
Verbraucher Zeitfenster werden nicht im Sunny Portal angemeldet

### DISCUSSION THREAD

**Discussion by SunnyHomeManager** [camueller/SmartApplianceEnabler#194] [Installation und Betrieb]
Normalerweise müsste doch ein im SAE konfiguriertes Zeitfenster im Portal angemeldet werden (heller Balken). Sol lange das aber nicht passiert, würde der Home Manager es auch nicht einplanen. Wo liegt denn da der Gedankenfehler?

![image](https://user-images.githubusercontent.com/99254790/152972990-3732f7a0-9f59-4f90-b3a7-b6bb20e73afc.png)

![image](https://user-images.githubusercontent.com/99254790/152973306-4ba494b7-1aa2-473f-8cea-3502d2ca0bd8.png)

![image](https://user-images.githubusercontent.com/99254790/152973479-c1a8e7ee-a395-4a65-9338-6120001596b3.png)

**Comment by camueller** [comment]
Entscheidend ist, was der SAE dem SHM meldet an der SEMP-Schnittstelle. Nur dann hat der SHM die Chance, den Bedarf einzuplanen. Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp3

Ich sehe gerade, das in Deinem Screenshot min > max eingegeben ist. Offensichtlich wird das im SAE nicht validiert und der SHM verwirft den Timeframe vermutlich deswegen.

BTW: Bist Du (bei) SMA - der Github-Account deutet darauf hin :-)

**Reply by SunnyHomeManager** [reply]
> Ich sehe gerade, das in Deinem Screenshot min > max eingegeben ist. Offensichtlich wird das im SAE nicht validiert und der SHM verwirft den Timeframe vermutlich deswegen.

Ich habe hier schon viele rum gestellt aber ich weiß einfach nicht ob es an mir liegt und wo man welche Parameter einstellen muss.

> BTW: Bist Du (bei) SMA - der Github-Account deutet darauf hin :-)

Ja, ... habe versucht einen Namen zu nehmen, der das auch aussagt aber Falko war schon belegt ;)

**Reply by camueller** [reply]
Hatte schon vermutet, dass Du es bist, nachdem Du gestern auch im PV-Forum geschrieben hattest :+1: 
Ich wusste gar nicht, dass Du selbst den SAE verwendest :-)
Falls Du Hilfe brauchst, können wir uns auch mal online zusammenschliessen ...

**Reply by SunnyHomeManager** [reply]
Nun ja, verwenden ist etwas zu viel gesagt.
Ich habe mir mal eine Umgebung aufgebaut und möchte einfach mehr verstehen was genau ihr hier im Detail macht und wenn ich die Zeit schon investiere, dann soll es ja auch laufen ;)

**Comment by SunnyHomeManager** [comment]
Heute wird es richtig eingeplant 

![image](https://user-images.githubusercontent.com/99254790/153132745-fd35bb2e-ee0d-4a56-9958-629aa9fbff88.png)

Das ist meine Konfiguration des Zeitplan:
![image](https://user-images.githubusercontent.com/99254790/153132869-dc1d0a9a-489f-41f6-aad7-c3fe1ba85c9d.png)
